proposals/description/cr_oracle_cleanup.ts

```diff
@@ -0,0 +1,152 @@
+      description:
+        'Revoke ORACLE_ADMIN_ROLE from collateralizationOracleGuardian (no more guardian updates of cached CR Oracle)'
+    },
+    {
+      target: 'tribeReserveStabilizer',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in Tribe Reserve Stabilizer (backstop) to the uncached CR Oracle'
+    },
+    {
+      target: 'pcvEquityMinter',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in PCV Equity Minter (buybacks) to the uncached CR Oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.volt, addresses.voltOracle],
+      description: 'Replace VOLT oracle in CR oracle'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.ethPSM, '10000'],
+      description: 'Withdraw remaining ETH/FEI liquidity from Uniswap v2'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.voltOracle],
+      description: 'Replace VOLT oracle in CR oracle'
+    },
+    /*{
+      target: 'raiPriceBoundPSM',
+      values: '0',
+      method: 'unpause()',
+      arguments: (addresses) => [],
+      description: 'Unpause RAI PSM'
+    },*/
+    /*{
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [addresses.ethTokemakPCVDeposit, addresses.ethPSM, '10000'],
+      description: 'Withdraw all ETH from Tokemak'
+    },*/
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.dpiToDaiLensDai, // 0$ left, auction ended
+          addresses.dpiToDaiLensDpi, // 0$ left, auction ended
+          addresses.bammDeposit, // 0$ left, deprecated (need redeploy)
+          addresses.rariPool7LusdPCVDeposit, // 0$ left
+          addresses.rariPool6FeiPCVDepositWrapper, // 114$ left
+          addresses.rariPool19FeiPCVDepositWrapper, // 0$ left
+          addresses.rariPool24FeiPCVDepositWrapper, // 6k$ left
+          addresses.agEurDepositWrapper, // no more agEUR on the timelock
+          addresses.wethDepositWrapper, // no more WETH on the timelock
+          addresses.d3poolConvexPCVDeposit, // 0$ left in Convex d3
+          addresses.d3poolCurvePCVDeposit, // 1034$ left in Curve d3
```

Comment:
there is also ~$15k FEI that should be withdrawn. tbh I'd include withdrawals for as many of these assets as possible:
- $15k FEI from d3Curve
- ~$1k FEI liquidity from d3Curve
- RAI in RAI psm
- FEI in RAI psm
- COMP in compoundETHPCVDepositWrapper

Reply:
Might as well transfer in the remaining AAVE and COMP from the 2 live deposits too per https://snapshot.fei.money/#/proposal/0xf789539597a01bae85be052405854751737562f0832aab78f8d450d8afcb7bc3